build.gradle

```diff
@@ -31,11 +34,21 @@ repositories {
 }
 
 dependencies {
-    testCompile 'org.junit:junit5-api:5.0.0-ALPHA'
+    testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+    testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-core:${log4JVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-jul:${log4JVersion}"
 }
 
-junit5 {
-    version '5.0.0-ALPHA'
+junitPlatform {
+    logManager 'org.apache.logging.log4j.jul.LogManager'
```

Comment:
Understood. And I learned [JUnit5 gradle sample](https://github.com/junit-team/junit5-samples/blob/master/junit5-gradle-consumer/build.gradle#L37) does not have `platformVersion` in `junitPlatform` block. Then We don't need `platformVersion` here.